model/src/main/java/org/cbioportal/model/Sample.java

```diff
@@ -54,6 +54,7 @@ public String toString() {
     private String cancerStudyIdentifier;
     private Boolean sequenced;
     private Boolean copyNumberSegmentPresent;
+    private Boolean fusionPresent;
```

Comment:
This property be true even if no fusion events were detected. So I request that this be renamed to profiledForFusions or profiledForStructuralVariants or analyzedForFusions or fusionScanPerformed.

fusionPresent suggests that a fusion was detected in the sample.

Reply:
go with profiledForFusions.